[2025-03-12T21:32:52.516+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-12T21:32:52.542+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_lake_pipeline.process_to_raw manual__2025-03-12T21:32:45.150662+00:00 [queued]>
[2025-03-12T21:32:52.555+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_lake_pipeline.process_to_raw manual__2025-03-12T21:32:45.150662+00:00 [queued]>
[2025-03-12T21:32:52.556+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-12T21:32:52.572+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): process_to_raw> on 2025-03-12 21:32:45.150662+00:00
[2025-03-12T21:32:52.579+0000] {standard_task_runner.py:72} INFO - Started process 199 to run task
[2025-03-12T21:32:52.584+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_lake_pipeline', 'process_to_raw', 'manual__2025-03-12T21:32:45.150662+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_dag.py', '--cfg-path', '/tmp/tmpt279jv5m']
[2025-03-12T21:32:52.585+0000] {standard_task_runner.py:105} INFO - Job 54: Subtask process_to_raw
[2025-03-12T21:32:52.607+0000] {logging_mixin.py:190} WARNING - /usr/local/lib/python3.10/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-12T21:32:52.646+0000] {task_command.py:467} INFO - Running <TaskInstance: data_lake_pipeline.process_to_raw manual__2025-03-12T21:32:45.150662+00:00 [running]> on host ***-scheduler
[2025-03-12T21:32:52.765+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_lake_pipeline' AIRFLOW_CTX_TASK_ID='process_to_raw' AIRFLOW_CTX_EXECUTION_DATE='2025-03-12T21:32:45.150662+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-12T21:32:45.150662+00:00'
[2025-03-12T21:32:52.767+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-12T21:32:52.785+0000] {manage_buckets.py:24} INFO - [Data manager] Downloading data from HuggingFace...
[2025-03-12T21:32:56.491+0000] {logging_mixin.py:190} WARNING - Generating test split:   0%|          | 0/4358 [00:00<?, ? examples/s]
[2025-03-12T21:32:56.516+0000] {logging_mixin.py:190} WARNING - Generating test split: 100%|##########| 4358/4358 [00:00<00:00, 183625.10 examples/s]
[2025-03-12T21:32:56.517+0000] {logging_mixin.py:190} WARNING - Generating train split:   0%|          | 0/36718 [00:00<?, ? examples/s]
[2025-03-12T21:32:56.619+0000] {logging_mixin.py:190} WARNING - Generating train split:  71%|#######   | 26000/36718 [00:00<00:00, 254902.56 examples/s]
[2025-03-12T21:32:56.641+0000] {logging_mixin.py:190} WARNING - Generating train split: 100%|##########| 36718/36718 [00:00<00:00, 295818.67 examples/s]
[2025-03-12T21:32:56.643+0000] {logging_mixin.py:190} WARNING - Generating validation split:   0%|          | 0/3760 [00:00<?, ? examples/s]
[2025-03-12T21:32:56.656+0000] {logging_mixin.py:190} WARNING - Generating validation split: 100%|##########| 3760/3760 [00:00<00:00, 296417.24 examples/s]
[2025-03-12T21:32:56.669+0000] {manage_buckets.py:27} INFO - [Data manager] Data successfully downloaded.
[2025-03-12T21:32:56.670+0000] {manage_buckets.py:30} INFO - [Data manager] Transferring data to S3 bucket...
[2025-03-12T21:32:56.671+0000] {aws_s3_repository.py:133} INFO - [AWS S3] Uploading into raw ...
[2025-03-12T21:32:56.672+0000] {aws_s3_repository.py:136} INFO - [AWS S3] Transferring train data...
[2025-03-12T21:32:56.672+0000] {aws_s3_repository.py:148} CRITICAL - [AWS S3] Error while uploading data into raw ...
[2025-03-12T21:32:56.673+0000] {manage_buckets.py:42} CRITICAL - [Data manager] Error during uploading files to S3 bucket named raw
[2025-03-12T21:32:56.673+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-12T21:32:56.692+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-12T21:32:56.693+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_lake_pipeline, task_id=process_to_raw, run_id=manual__2025-03-12T21:32:45.150662+00:00, execution_date=20250312T213245, start_date=20250312T213252, end_date=20250312T213256
[2025-03-12T21:32:56.761+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-12T21:32:56.804+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-12T21:32:56.809+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
